include/SFML/Audio/SoundStream.hpp

```diff
@@ -281,7 +281,8 @@ class SFML_AUDIO_API SoundStream : public SoundSource
 
     enum
     {
-        BufferCount = 3 ///< Number of audio buffers used by the streaming loop
+        BufferCount = 3,    ///< Number of audio buffers used by the streaming loop
+        BufferRetries = 2   ///< Number of retries (excluding initial try) for onGetData()
```

Comment:
> Maybe you remember why you put it in the header?

Sure. It was because this constant has a direct impact on the user's code, and thus can be useful to know.  Another reason is that it is used just below, to declare the m_buffers and m_endBuffers arrays ;)